test/list/manage/components/open-faq-test.js

```diff
@@ -0,0 +1,25 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import chai, { expect } from "chai";
+import React from "react";
+import sinon from "sinon";
+import sinonChai from "sinon-chai";
+
+import mountWithL10n from "test/mocks/l10n";
+import OpenFAQ from
+       "src/webextension/list/manage/components/open-faq";
+
+chai.use(sinonChai);
+
+describe("list > manage > components > <OpenFAQ/>", () => {
+  it("FAQ link opened", () => {
+    const windowOpen = sinon.stub(window, "open");
```

Comment:
If I understand correctly (and I rarely do), this is causing Travis to fail because we're doing a `sinon.stub(window, "open")` for both the feedback button tests for the Feedback and now FAQ button: https://github.com/mozilla-lockbox/lockbox-extension/blob/ed488b5317ca87d84f618a6bf32d29c6de54eda0/test/list/manage/components/send-feedback-test.js#L18

> "Attempted to wrap open which is already wrapped"

Not sure if we need a cleanup step to unstub this for each suite (if that's a thing). ¯\\\_(ツ)\_/¯

---

https://travis-ci.org/mozilla-lockbox/lockbox-extension/jobs/312183180#L5109-L5122

```
FAILED TESTS:
  list > manage > components > <SendFeedback/>
    ✖ feedback link opened
      Firefox 57.0.0 (Linux 0.0.0)
    Attempted to wrap open which is already wrapped
    checkWrappedMethod@webpack:///node_modules/sinon/lib/sinon/util/core/wrap-method.js:39:0 <- test/index.js:66490:21
    wrapMethod@webpack:///node_modules/sinon/lib/sinon/util/core/wrap-method.js:85:0 <- test/index.js:66536:13
    stub@webpack:///node_modules/sinon/lib/sinon/stub.js:60:0 <- test/index.js:35429:44
    @webpack:///test/list/manage/components/send-feedback-test.js:18:23 <- test/index.js:241843:24
    
    --------------
    wrapMethod@webpack:///node_modules/sinon/lib/sinon/util/core/wrap-method.js:110:0 <- test/index.js:66561:26
    stub@webpack:///node_modules/sinon/lib/sinon/stub.js:60:0 <- test/index.js:35429:44
    @webpack:///test/list/manage/components/open-faq-test.js:18:23 <- test/index.js:241550:24
```

https://travis-ci.org/mozilla-lockbox/lockbox-extension/jobs/312183180#L4836-L4839

```
  list > manage > components > <OpenFAQ/>
    ✔ FAQ link opened
  list > manage > components > <SendFeedback/>
    ✖ feedback link opened
```

Reply:
Thanks, @pdehaan. I did see the failure but chose not to fix it until I knew if we should continue down this path.

Tests are passing again based on @jimporter's new telemetry middware approach now. ✅